lib/zxcvbn_tester.rb

```diff
@@ -0,0 +1,10 @@
+module Zxcvbn
+  class Tester
```

Comment:
This is overriding a class in a gem that we use. I suggested we include here because if that. With this knowledge, where do you think we should include, @zachmargolis ?